golem-worker-service-base/src/gateway_rib_compiler/mod.rs

```diff
@@ -45,6 +45,11 @@ impl WorkerServiceRibCompiler for DefaultWorkerServiceRibCompiler {
                     path: Path::from_elems(vec!["headers"]),
                     inferred_type: InferredType::Str,
                 },
+                GlobalVariableTypeSpec {
+                    variable_id: VariableId::global("request".to_string()),
+                    path: Path::from_elems(vec!["header"]),
+                    inferred_type: InferredType::Str,
+                },
```

Comment:
I keep bumping into this problem. It's very hard to say which one is more intuitive. 
`request.path` or `request.query` or `request.body` don't have this singular plural confusion - `request.header.auth-bearer` or `request.headers.auth-bearer`. May be `header` is more correct. But I am pretty sure users will need to keep coming back to docs to re-assert themselves.